### DISCUSSION TITLE
Οριζόντια παράταση για την παράδοση της ΓΕ3

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60/syllabus#59] [Q&A] [answered]
Καλημέρα σας και καλή χρονιά να έχουμε, με υγεία και δημιουργικότητα!

Θα ήθελα να ρωτήσω αν είναι δυνατόν να δοθεί σε όλους τους φοιτητές οριζόντια παράταση 1 εβδομάδας για την παράδοση της ΓΕ3, η οποία κανονικά λήγει την 1η Φεβρουαρίου.  Αυτή η παράταση δεν προκαλεί πρόβλημα μετακύλισης στις επόμενες ημερομηνίες παράδοσης, οι οποίες είναι αρκετά αργότερα (ΓΕ4: 22 Μαρτίου και ΓΕ5: 3 Μαΐου).

Σας ευχαριστώ

**Comment by dimidimots** [comment]
Καλησπέρα καλή χρονιά, και εμένα θα με βόλευε φοβερά...

**Comment by Katethoma** [comment]
Καλησπέρα. Χρόνια Πολλά και Καλή Χρονιά ! Κι εγώ συμφωνώ για την παράταση.

**Comment by epidrome** [comment, marked as answer]
OK!
